service_mesh/v2x/ossm-observability.adoc

```diff
@@ -20,6 +20,14 @@ include::modules/ossm-kiali-viewing-metrics.adoc[leveloffset=+2]
 
 include::modules/ossm-distr-tracing.adoc[leveloffset=+1]
 
+include::modules/ossm-configuring-distr-tracing-tempo.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc[Installing the distributed tracing platform (Tempo)].
```

Comment:
```suggestion
xref:../../distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc[Installing the distributed tracing platform (Tempo)]
```
No period on these unless a complete sentence (generally that isn't _all_ title). Again, not holding merge on this.

Reply:
Fixed in different PR